web/styles/recent_view.css

```diff
@@ -4,550 +4,528 @@
     overflow: hidden;
     display: flex;
     flex-direction: column;
-    max-height: 100vh;
+    position: sticky;
+    top: var(--navbar-fixed-height);
+    z-index: 1;
+}
 
-    #recent_view_table {
-        max-width: 100%;
-        overflow: hidden !important;
-        display: flex;
-        flex-direction: column;
-        border-style: solid;
-        border-color: hsl(0deg 0% 88%);
-        border-width: 0 1px;
-        border-radius: 0;
-        /* To make the table span full height
-         * when rows don't reach bottom of the
-         * window. This makes the border span
-         * fully to bottom in that case.
-         */
-        min-height: 100vh;
-
-        & td {
-            vertical-align: middle;
-            padding: 3px 8px;
-        }
+.recent_view_container #recent_view_table {
+    max-width: 100%;
+    overflow: hidden !important;
+    display: flex;
+    flex-direction: column;
+    border: 0;
+}
 
-        .recent_view_focusable {
-            /* Use flexbox to align icons vertically */
-            display: flex;
-            align-items: center;
+#recent_view_table .table,
+#recent-view-content-table {
+    /* To keep border properties to the thead th. */
+    border-collapse: separate;
 
-            .filter-icon {
-                /* Maintain righthand space between icon
-                   and stream name. */
-                margin-right: 3px;
-            }
+    border-spacing: 0;
+    width: 100%;
+}
 
-            & > * {
-                outline: 0;
-            }
+#recent_view {
+    display: none;
+    padding-top: var(--navbar-fixed-height);
+    /*
+    Add padding bottom equal to `#bottom-whitespace`. This helps us keep
+    #compose visible at its max-height without overlapping with any visible
+    topics.
+
+    Alternative is to adjust the max-height of `table_fix_head` based on compose height which is an
+    expensive repaint. The downside of not doing so is that the scrollbar is not visible to user when
+    user is at the bottom of scroll container when the compose box is open.
```

Comment:
Can you post a screenshot of the problem with this implementation on this detail?

Reply:
Removed that comment since it doesn't help to think about it when looking at this CSS.